External.CustomerReviewsModule/External.CustomerReviewsModule/External.CustomerReviewsModule.Core/Services/ICustomerReviewService.cs

```diff
@@ -0,0 +1,20 @@
+using External.CustomerReviewsModule.Core.Models;
+using External.CustomerReviewsModule.Core.Models.Search;
+
+namespace External.CustomerReviewsModule.Core.Services
+{
+    public interface ICustomerReviewService
+    {
+        CustomerReview[] GetByIds(string[] ids);
+
+        IdentifierModel CreateCustomerReview(CustomerReviewCreateModel customerReviewCreateModel);
+
+        void UpdateCustomerReview(string id, CustomerReviewUpdateModel customerReviewUpdateModel);
+
+        void DeleteCustomerReviews(string id);
+
+        CustomerReviewResponseModel GetCustomerReviewById(string id);
+        void SaveCustomerReviews(CustomerReview[] customerReview);
```

Comment:
Please remove signatures of methods which you don't use from the outside of the service